src/components/AstroPiModel/AstroPiModel.scss

```diff
@@ -12,6 +12,7 @@
     display: flex;
     flex-flow: column;
     max-height: 50%;
+    min-height: 200px;
```

Comment:
I've used `px` here as both `rem` and `em` values were being affected by other css elements that continued to shrink the model.